src/applications/simple-forms/mock-simple-forms-patterns/pages/chapterSelect.js

```diff
@@ -25,35 +25,37 @@ export default {
       title: 'Chapters to include in this form',
       required: true,
       labels: {
-        textInput: 'Text input',
-        numberInput: 'Number input',
-        formsPattern: 'Forms pattern',
+        arrayMultiPageAggregate: 'Array Multi-Page Aggregate',
```

Comment:
I think it would be best not to sort this one, since this represents the actual order in the form flow.

Reply:
Good catch, fixed! 👍